chapters/missing-data.qmd

```diff
@@ -0,0 +1,598 @@
+---
+knitr:
+  opts_chunk:
+    cache.path: "../_cache/missing-data/"
+---
+
+# Missing Data {#sec-missing-data}
+
+```{r}
+#| label: missing-data-setup
+#| include: false
+#| message: false
+#| warning: false
+
+source("../R/_common.R")
+
+# ------------------------------------------------------------------------------
+# Required packages
+
+library(tidymodels)
+library(rpart)
+library(partykit)
+
+# ------------------------------------------------------------------------------
+# Set options
+
+tidymodels_prefer()
+theme_set(theme_transparent())
+set_options()
+
+# ------------------------------------------------------------------------------
+# Load other data
+
+load("../RData/ames_original.RData")
```

Comment:
Can you put the code to make `ames_original` in the `R` directory?

Reply:
Done.  I added AmesHousing.xls to the delimited folder and make_ames_original_data.R to the R folder in the main branch.